bin/dart_doc_syncer.dart

```diff
@@ -3,22 +3,40 @@ import 'dart:async';
 import 'package:logging/logging.dart';
 
 import 'package:dart_doc_syncer/documentation_updater.dart';
+import 'package:dart_doc_syncer/example2uri.dart';
+import 'package:dart_doc_syncer/options.dart';
 
 /// Syncs an example application living in the angular.io repository to a
 /// dedicated repository that will contain a generated cleaned-up version.
 ///
-///     dart_doc_syncer <examplePath> <exampleRepository>
-Future main(List<String> args) async {
-  Logger.root.level = Level.WARNING;
+///  dart_doc_syncer [-h|-n|-v] [<exampleName> | <examplePath> <exampleRepo>]
+Future main(List<String> _args) async {
```

Comment:
Notice that the next line defines a var named `args` which matches the value that `args` had before support for flags was added. Prefixing the function parameter name with `_` and then assigning to `args` allows me to leave the rest of the code unchanged (regarding the use of `args`).